src/object/srv_obj.c

```diff
@@ -2951,6 +2977,9 @@ ds_obj_rw_handler(crt_rpc_t *rpc)
 	/* Execute the operation on all targets */
 	rc = dtx_leader_exec_ops(dlh, obj_tgt_update, NULL, 0, &exec_arg);
 
+	if (max_ver < dlh->dlh_rmt_ver)
+		max_ver = dlh->dlh_rmt_ver;
```

Comment:
could you please explain a little bit why add the "max_ver" checks in this func? thanks.

Reply:
Because different DTX participants may have different pool map versions, here we collect the max version from all the other participants and reply it back to client when version unmatched.